### ISSUE TITLE
Incorrect predection/detection of the given input

### ISSUE THREAD

**Issue by shrimad-mishra-cognoai** [AI4Bharat/IndicLID#4]
Hi, I have been exploring the model but encountered some issues:-

If you provide: Hi kya hal hai aapka then it sometimes predicts it as pan_lat, snd_lat and eng_lat.

If you provide: Hi how are you then it sometimes predicts it as o, snd_lat and eng_lat, ori_Latn, and other.

Can you please provide any solution for this? I think this is a bit biased towards the Latin languages.

**Comment by chandan-wiai** [comment]
Yes, I observed the same when I ran the example colab [notebook](https://colab.research.google.com/drive/1pLMeaGhYgfNRmYHPHkvAmcR-8xMMZOme?usp=sharing) with different examples of Hindi written in Latin. 
@yashmadhani97 Are we doing something incorrect?
